test-config.yaml

```diff
@@ -0,0 +1,6 @@
+providers: 
```

Comment:
@C123R  Can you plz remove this? apiKeys should never be submitted on public repo.

Reply:
@usamaahmadkhan It is just a random key :)